### PR TITLE
Fix php unit test cases

### PR DESCRIPTION
**Pull request by sunilagaikwad**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     },
     "autoload": {
         "psr-0": {
-            "TS\\Writer\\": "src/"
+            "TS\\Writer\\": "src/",
```
Comment (eastern-ravindra) [inline composer.json:23]: This namespace also needs to be renamed then?

Comment (sunilagaikwad) [reply composer.json:23]: For source, we will do it of course but not here. We may do that on least priority

File: tests/AcsiTswriterTest/FactoryTest.php [hunk 2]
```diff
@@ -77,22 +73,21 @@ public function testRegistrationWithInstance()
...
+//    public function testFactory(): void
+//    {
+//        $instance = new Txt(new EventDispatcher());
+//
+//        $this->factory->registerWriter($instance);
+//
+//        $writer = $this->factory->createForType('txt');
+//
+//        $this->assertEquals($instance, $writer);
+//    }
```
Comment (eastern-ravindra) [inline tests/AcsiTswriterTest/FactoryTest.php]: Why commented code?

Comment (sunilagaikwad) [reply tests/AcsiTswriterTest/FactoryTest.php]: oops ..thats not intentionally . may be added to test something and then forgot to update it.

**Review by eastern-ravindra — APPROVED** [review]

**Pull request merged**